mappings/net/minecraft/network/packet/s2c/play/EntitySoundPlayS2CPacket.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/unmapped/C_esombqse net/minecraft/network/packet/s2c/play/PlaySoundFromEntityS2CPacket
+CLASS net/minecraft/unmapped/C_esombqse net/minecraft/network/packet/s2c/play/EntitySoundPlayS2CPacket
```

Comment:
This name may imply that the played sound is an entity sound, instead of saying that it's just played from the entity